jukebox/utils/dist_adapter.py

```diff
@@ -0,0 +1,92 @@
+import torch.distributed as dist
+from enum import Enum
+
+class ReduceOp(Enum):
+    SUM = 0,
+    PRODUCT = 1,
+    MIN = 2,
+    MAX = 3,
+    BAND = 4, # Bitwise AND
+    BOR = 5, # Bitwise OR
+    BXOR = 6, # Bitwise XOR
+
+    def ToDistOp(self):
+        return {
+            self.SUM: dist.ReduceOp.SUM,
+            self.PRODUCT: dist.ReduceOp.PRODUCT,
+            self.MIN: dist.ReduceOp.MIN,
+            self.MAX: dist.ReduceOp.MAX,
+            self.BAND: dist.ReduceOp.BAND,
```

Comment:
nitpick: We provide an option to use fused_adam with pytorch 1.1.0 and apex, but bitwise ops are not supported in 1.1.0. Since these ops are not used in this repo, could we remove them for now?

Reply:
sure thing